src/search.cpp

```diff
@@ -110,10 +111,33 @@ setup_queryParser(Xapian::QueryParser* queryparser,
             stopper->add(stopWord);
         }
         stopper->release();
-        queryparser->set_stopper(stopper);
+        queryParser->set_stopper(stopper);
     }
 }
 
+/*
+ * subquery_phrase: selects documents that have the terms in the order of the query
+ * within a specified window.
+ * subquery_and: selects documents that have all the terms in the query.
+ * subquery_phrase by itself is quite exclusive. To include more "similar" docs,
+ * we combine it with subquery_and using OP_OR operator. If a perticular document
+ * has a weight of A in subquery_phrase and B in subquery_and, the net weight of
+ * that document becomes A+B. So the documents closer to the query gets a higher.
+ */
+Xapian::Query parseQuery(Xapian::QueryParser* queryParser, std::string qs, int flags, std::string prefix, bool suggestion_mode) {
```

Comment:
Why we have `queryParser` in camel case and `suggestion_mode` without camel case? This problem appears in many other places of this PR.

Reply:
Changing the variables I used in the patch to underscore casing. It's worth noting that we still have a mix of camel case and underscore throughout the file.